### PR TITLE
Add guidance to Installation section

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Addresses https://github.com/blackbaud/skyux2-docs/issues/341. Also cleans up a handful of component topics where the module and/or package were not displaying correctly. (There are several others I wasn't able to fix here. I will create a separate issue for those after this update.)

### CODE REVIEW

File: src/app/components/shared/demo-page.component.html [hunk 1]
```diff
@@ -26,7 +26,13 @@ <h4 *ngIf="skyuxModulesForDisplay">
     <stache-page-anchor>
       Installation
     </stache-page-anchor>
+    <p>
+      To use <span *ngFor="let moduleName of skyuxModulesForDisplay"><stache-code>{{ moduleName }}</stache-code></span>, you must first install the module's NPM package to set it up as a dependency. In the command prompt, run the following command from your SPA's root directory:
```
Comment (Blackbaud-BobbyEarl) [inline src/app/components/shared/demo-page.component.html]: I'm wondering if we could clarify that you use exports from the module, not the module itself.  Things like pipes, components, and services. 

Perhaps `To use any exports from the {{ skyuxModulesForDisplay }} module, you must install it as a dependency to your project.  In the command prompt, run the following command in your SPA's root directory:`

File: src/app/components/shared/demo-page.component.html [hunk 2]
```diff
@@ -26,7 +26,13 @@ <h4 *ngIf="skyuxModulesForDisplay">
...
+    </p>
     <stache-code-block [code]="npmInstall"></stache-code-block>
+    <p>
+      After the module is in place, use an Angular <stache-code>app.module.ts</stache-code> file to make it available to your SPA. For SKY UX SPAs, add <span *ngFor="let moduleName of skyuxModulesForDisplay"><stache-code>{{ moduleName }}</stache-code></span> as both an import and an export in the <stache-code>app-extras.module.ts</stache-code> file that controls the rendering of the SPA. For SKY UX libraries, add <span *ngFor="let moduleName of skyuxModulesForDisplay"><stache-code>{{ moduleName }}</stache-code></span> as an import in the module for the component that consumes it.
```
Comment (Blackbaud-BobbyEarl) [inline src/app/components/shared/demo-page.component.html]: I'd like to see if we can make the guidance more generic, but then be explicit about what to do when using our SDK.  It is also only necessary to add it to the `exports` section.  Perhaps something like...

"After installation, you need to make any exports from the module available to your SPA by adding it to the `exports` section of your SPA's main module.  If you are using SKY UX SDK, add it to the exports section of your `app-extras.module.ts` file."

File: src/app/components/shared/demo-page.component.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/components/shared/demo-page.component.html]: Makes sense.

File: src/app/components/shared/demo-page.component.html (hunk 2 above)
Comment (blackbaud-johnly) [reply src/app/components/shared/demo-page.component.html]: Sounds good.

**Comment by blackbaud-johnly** [comment]
@Blackbaud-BobbyEarl, I made updates to address your feedback. Please just let me know if you see anything else.

**Comment by Blackbaud-BobbyEarl** [comment]
I think getting https://github.com/blackbaud/skyux2-docs/pull/383 in first will simplify some of the nasty logic we have to support in the template, and clean up this PR slightly.

**Comment by blackbaud-johnly** [comment]
Marked as hold pending https://github.com/blackbaud/skyux2-docs/pull/398.

**Pull request merged**